plugins/source/digitalocean/codegen/main.go

```diff
@@ -131,7 +131,7 @@ var Resources = []*recipes.Resource{
 		ExtraColumns: []codegen.ColumnDefinition{
 			{
 				Name:     "id",
-				Type:     schema.TypeString,
```

Comment:
This seems correct, but I think the fix should be to drop these custom definitions and use a custom caser, for example https://github.com/cloudquery/cloudquery/blob/9d5959985b62ebb29f56df61efba31a4fed893cf/plugins/source/aws/codegen/recipes/base.go#L78

Reply:
maybe we can do this in a follow-up PR as this PR is already quite huge